packages/core/src/common/quick-pick-service.ts

```diff
@@ -14,6 +14,8 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+import { HideReason } from '../browser';
```

Comment:
This is incorrect; you should not use anything from `browser` in `common.`

Reply:
Could I move it in packages/core/src/common/quick-pick-service.ts ?

Reply:
👍

Reply:
Done.